soepy/shared/shared_auxiliary.py

```diff
@@ -71,7 +73,14 @@ def calculate_log_wage_systematic(model_params, states, covariates):
     # Construct wage components
     gamma_0s = np.array(model_params.gamma_0s)[educ_level]
     gamma_1s = np.array(model_params.gamma_1s)[educ_level]
-    period_exp_sum = exp_p * np.array(model_params.g_s)[educ_level] + exp_f
+
+    if expected is True:
```

Comment:
A simple `if expected:` suffices here. Let's just do a short `if ... else .. ` construct here. No need to cover the case of passing in the wrong value. If so, then it will throw an error anyway as `period_exp_sum` is not defined.

Reply:
Done